template/page-dev-tools.html.jinja

```diff
@@ -0,0 +1,193 @@
+{% extends "base.html.jinja" %}
+
+{% block bodyclasses %}no-sidebar{% endblock %}
+{% block mainclasses %}page-dev-tools{% endblock %}
+
+{% block main %}
+
+<div class="">
+  <section class="py-20">
+    <div class="mx-auto text-lg-left">
+      <div class="d-flex flex-column-reverse">
+        <p class="mb-0">
+          {% trans %}Use the developer tools to test, explore, and validate XRP Ledger
+          API requests and behavior. {% endtrans %}
+        </p>
+        <h3 class="eyebrow mb-3"> {% trans %} Dev Tools  {% endtrans %}</h3>
+      </div>
+    </div>
+  </section>
+
+
+  
+  <section class="py-10">
+    <div class="mx-auto text-lg-left">
+      <ul class="nav nav-tabs pb-15" id="myTab" role="tablist">
+        <li class="nav-item" role="presentation">
+          <button class="nav-link active dev-tools-tab" id="explorers-tab" data-toggle="tab" data-target="#explorers" role="tab" aria-controls="explorers" aria-selected="true" >Explorers</button>
```

Comment:
The text for the tabs should be in `{% trans %}` blocks so that it can be translated.

```suggestion
          <button class="nav-link active dev-tools-tab" id="explorers-tab" data-toggle="tab" data-target="#explorers" role="tab" aria-controls="explorers" aria-selected="true" >{% trans %}Explorers{% endtrans %}</button>
```